plugins/gaming.py

```diff
@@ -119,21 +119,89 @@ def dice(text, notice):
     return "{} ({})".format(total, ", ".join(rolls))
 
 
+choice_split_re = re.compile(r'\s*(?:, ?or|,|or)\s*', re.I)  # regex for splitting choices for choose command
```

Comment:
While testing, `choose a, bor, c d or d` returned an empty string, might want to add a basic unit test function to the plugin, like
```python
def test_regex_split():
    assert choice_split_re.split('a or b') == ['a', 'b']
    ...
```

Reply:
If the function has the name `test_*`, the test suite will run it automatically